src/main/java/core/basesyntax/handlers/BalanceHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.handlers;
+
+import core.basesyntax.db.FruitStorage;
+
+public class BalanceHandler implements OperationHandler {
+    @Override
+    public void calculateResult(String fruit, int quantity) {
+        if (quantity < 0) {
+            throw new RuntimeException("Balance value can't be negative");
+        } else {
+            FruitStorage.storageData.put(fruit,quantity);
+        }

```

Comment:
We do not need else block in the end of method, it doesn't produce any difference in logic
```suggestion
    public void calculateResult(String fruit, int quantity) {
        if (quantity < 0) {
            throw new RuntimeException("Balance value can't be negative");
        }
        FruitStorage.storageData.put(fruit,quantity);
```
Please recheck similar cases